oximeter/collector/Cargo.toml

```diff
@@ -6,7 +6,7 @@ description = "The oximeter metric collection server"
 license = "MPL-2.0"
 
 [dependencies]
-dropshot = { git = "https://github.com/oxidecomputer/dropshot", branch = "main" }
+dropshot = { git = "https://github.com/oxidecomputer/dropshot", branch = "main", features = [ "usdt-probes" ] }
```

Comment:
If oxidecomputer/dropshot#208 fixes this, I think that's preferable to changing all these Cargo.toml files.

Reply:
That seems right, that we'd prefer to fix this via `dropshot`. But we _do_ want these probes enabled at some point, right? I can do that in a separate PR, if that's preferable, after bringing in https://github.com/oxidecomputer/dropshot/pull/208. Thoughts.

Reply:
Oh, that's a good point.  I'd forgotten about that.  Yeah, let's just leave this in (and maybe mention in a comment or the PR description that we're _also_ deliberately enabling the probes as part of this).

Reply:
Yep, will do. Thanks for the close look over.